### ISSUE TITLE
Architecture names in goreleaser

### ISSUE THREAD

**Issue by ncw** [goreleaser/nfpm#21]
I've successfully managed to make .deb and .rpm for the [rclone release](https://rclone.org/downloads/) with nfpm - hooray and thank you!

However I had a report that my [386 package](https://downloads.rclone.org/v1.40/rclone-v1.40-linux-386.deb) wouldn't install.

I think that is because I put the architecture in as 386 (go style) whereas I think debian requires i386.

I can translate them easily enough, but then I was wondering if this should be something nfpm should do.  I was also wondering if it affects goreleaser.

These are the translations I came up with when thinking about it

```go
// Goarch to debian arch mapping
var goarchToDebian = map[string]string{
       "386":    "i386",
       "amd64":  "amd64",
       "arm":    "armhf",
       "arm64":  "arm64",
       "mips":   "mips",
       "mipsle": "mipsel",
}
```

What do you think?

**Comment by caarlos0** [comment]
hmm, probably... on the rpm packager there is a fix for amd64 or something already... I think we can do the same for the deb :)

**Comment by caarlos0** [comment]
check #22

**Comment by ncw** [comment]
Nice one - thank you very much :-)

I'll give this a go and report back

**Comment by caarlos0** [comment]
fixed on #22

**Comment by caarlos0** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
